components/LinkCollective.js

```diff
@@ -40,7 +40,7 @@ const LinkCollective = ({
   const { slug, name } = collective;
   const link = (
     <Link
-      href={getCollectivePageRoute(collective)}
+      href={props.onClick ? '#' : getCollectivePageRoute(collective)}
```

Comment:
:no_entry_sign: A link without href should be a button, not a link. The standard approach is:

```jsx
<Button variant="link">
```

Reply:
You're right, this is a leftover of a prototype that I didn't shipped (with StackedAvatars).